infra/terraform/README.md

```diff
@@ -80,14 +105,14 @@ Deployment can then be done by issuing the following command after initializing
 ```console
 $ terraform workspace show
 prod
-$ terraform apply --var-file prod-usea1.tfvars --var-file prod.secrets.tfvars
+$ terraform apply --var-file prod-usea1.tfvars
 ..
 ```
 
-Note that it may be required to disable refreshing state for subsequent applies (because of the Postgres SSH tunnel. Data sources are deferred to the apply phase usually).
+Note that it may be required to disable refreshing state for subsequent applies (because of the Postgres SSH tunnel. Data sources are deferred to the apply phase, usually).
 
 ```console
-$ terraform apply --var-file prod-usea1.tfvars --var-file prod.secrets.tfvars -refresh=false
+$ terraform apply --var-file prod-usea1.tfvars -refresh=false
```

Comment:
```suggestion
$ terraform apply --var-file prod-usea1.tfvars --refresh=false
```

Reply:
Huh interesting that both `--` and `-` are valid. I'll apply the change for consistency

Reply:
Oh, I thought it was a typo, didn't know that `-refresh` also works 😄